native-tokens/README.md

```diff
@@ -12,15 +13,15 @@
     # ...
    ] ++ (lib.optionals (!stdenv.isDarwin) [ rPackages.plotly R ]));
    ```
-1. Change to `plutus` directory and run `nix-shell`. Run the following steps inside `nix-shell`.
-2. Change to `plutus-semantics/native-tokens/native-tokens`.
-3. Run `cabal build`.
-4. Run `cabal run native-tokens-scripts out_scripts scripts`.
-5. There will be 5 flat format files in `out_scripts`.
-   In bash, run the following command. (Make sure 
-   `.../plutus/dist-newstyle/build/x86_64-linux/ghc-8.10.4.20210212/plutus-core-0.1.0.0/x/uplc/build/uplc/uplc` is in your `PATH`.)
+1. Change to `native-tokens/native-tokens`.
+1. Create a symbolic link from `plutus/shell.nix` to `native-tokens/native-tokens`.
+1. Run `nix-shell --command 'cabal build'`.
+1. Run `nix-shell --command 'cabal run native-tokens-scripts out_scripts scripts'`.
+1. There will be 5 flat format files in `out_scripts`.
+1. In bash, run the following command. (Make sure 
+   `plutus/dist-newstyle/build/x86_64-linux/ghc-8.10.4.20210212/plutus-core-0.1.0.0/x/uplc/build/uplc/uplc` is in your `PATH`.)
```

Comment:
Is there some canonical way to get the plutus binaries onto PATH with nix? I just did a manual `export PATH...` command the last time I followed these instructions.

Reply:
IDK